pytorch_lightning/loggers/tensorboard.py

```diff
@@ -37,7 +38,6 @@ class TensorBoardLogger(LightningLoggerBase):
         \**kwargs: Other arguments are passed directly to the :class:`SummaryWriter` constructor.
 
     """
-    NAME_CSV_TAGS = 'meta_tags.csv'
```

Comment:
I think it would be good to have it as a constant somewhere instead of hard-coding the string.

Reply:
done aacc074983c67837c52ab2de65143e8d6285103a